packages/ckeditor5-link/tests/linkimageediting.js

```diff
@@ -11,6 +11,13 @@ import normalizeHtml from '@ckeditor/ckeditor5-utils/tests/_utils/normalizehtml'
 import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
 import ImageCaptionEditing from '@ckeditor/ckeditor5-image/src/imagecaption/imagecaptionediting';
 
+const linkIconInditatorElement = '<span class="ck ck-link-image_icon"><svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">' +
```

Comment:
That can be an import from the SVG file (the \<svg> part) so you won't need to update it when the SVG changes.

Reply:
🤦 oh, right...

Reply:
Fixed it, but with additional dev comment.